security/putty/Portfile

```diff
@@ -19,28 +20,13 @@ homepage            http://www.chiark.greenend.org.uk/~sgtatham/putty/
 master_sites        ftp://ftp.chiark.greenend.org.uk/users/sgtatham/putty-${version}/ \
                     http://the.earth.li/~sgtatham/putty/${version}/
 
-checksums           rmd160  9357dfca867dc91be09c735b62025f9c614ab46d \
-                    sha256  bb8aa49d6e96c5a8e18a057f3150a1695ed99a24eef699e783651d1f24e7b0be
-
-worksrcdir          ${worksrcdir}/unix
+checksums           rmd160  d4e27e59e3a7235089ccae854db2507a650da02f \
+                    sha256  419a76f45238fd45f2c76b42438993056e74fa78374f136052aaa843085beae5 \
+                    size    2619481
 
 depends_build       port:pkgconfig
-
-# unix configure calls the configure in top folder
-patch.post_args     -d ${worksrcpath}/../
-patchfiles          patch-configure.diff
-
-configure.args-append     --without-gtk
-configure.cppflags-delete -I${prefix}/include
-configure.cppflags-append -isystem${prefix}/include
-
-variant gui description {Build the GUI tools (including putty itself)} {
-    depends_lib-append    path:lib/pkgconfig/gtk+-2.0.pc:gtk2
-    configure.args-replace --without-gtk --with-gtk=2
-    require_active_variants gtk2 x11
-}
-
-default_variants +gui
+depends_lib         path:lib/pkgconfig/gtk+-2.0.pc:gtk2
+require_active_variants gtk2 x11
```

Comment:
why did you remove the variant?

Reply:
Done

Reply:
not sure, is there no CMake configure argument to do so?

Reply:
Looking at cmake/gtk.cmake - the port just tries to find GTK by using pkgconfig or gtk-config

Reply:
I found a way to build without GTK

Reply:
I see it indeed appears there is no configure flag to completely disable it... It was anyway a default variant in the past so unless people specifically disabled the GTK interface they would get it - so I think it's fine. 

The only thing I noticed by typing `ccmake` in the directory is that there is a `PUTTY_GTK_VERSION` flag that by default is set to `ANY` but one can also specify `1`, `2`, and `3`. I think it would be good to explicitly set this to make sure the builds will be reproducible (so set it to `3` if we're going to depend on the `gtk3` port). Since one can set different version for GTK one could add `gtk2` and `gtk3` as variants and then, for example, add `gtk3` as a default variant. But again, I am not sure if that is really of interest to users to be able to select a version.

Reply:
My latest changes include setting PUTTY_GTK_VERSION explicitly